MainWindow.py

```diff
@@ -0,0 +1,59 @@
+# File: MainWindow.py
+
+import gi
+from PkgDataProvider import PkgDataProvider
+
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
+
+class MainWindow(Gtk.Window):
```

Comment:
The class should start two lines after the imports; the same goes for the other files.